packages/pyright-internal/src/analyzer/docStringConversion.ts

```diff
@@ -421,8 +424,10 @@ class DocStringConverter {
     }
 
     private _beginBacktickBlock(): boolean {
-        if (this._currentLine().startsWith('```')) {
-            this._appendLine(this._currentLine());
+        const match = this._currentLine().match(CodeBlockStartRegExp);
+        if (match !== null) {
+            this._blockIndent = this._currentIndent()
+            this._appendLine('```' + match[1]) // remove indentation, preserve language tag
```

Comment:
Nit: Don't use end-of-line comments. Move to previous line. Comments should begine with capital letter and end in period if they are complete sentence.